ImperatorToCK3/Data_Files/configurables/province_mappings.txt

```diff
@@ -888,7 +888,7 @@
 	link = { comment = "# North & West Poland" }
 	link = { imp = 4002 ck3 = 3067 ck3 = 3065 ck3 = 3064 } # Budorigum -> SPROTTAU, BYTOM, GRUNBERG
 	link = { imp = 4887 ck3 = 527 ck3 = 4965 } # Cracovia -> Krakow, Tarnow
-	link = { imp = 6282 ck3 = 3201 } # Cureta -> Mohrungen 
+	link = { imp = 6282 ck3 = 3201 } # Cureta -> Mohrungen

```

Comment:
This file is generated by our ProvinceMapper tool, so your changes will probably get overwritten.